thrust/testing/catch2_test_helper.h

```diff
@@ -66,6 +66,10 @@ using integral_vector_list = cuda::std::__type_list<
   thrust::universal_vector<int>,
   thrust::universal_host_pinned_vector<int>>;
 
+// corresponds to DECLARE_GENERIC_UNITTEST
+using generic_list =
+  cuda::std::__type_list<signed char, unsigned char, short, unsigned short, int, unsigned int, float>;
+
 // corresponds to DECLARE_VARIABLE_UNITTEST
 using variable_list =
   cuda::std::__type_list<signed char, unsigned char, short, unsigned short, int, unsigned int, float, double>;
```

Comment:
I am wondering whether we shouldnt just also test `double`

Reply:
Oh, I did so as well! The generic and variable tests are almost the same. I believe there is no good reason to not have `double`.